dwave/system/samplers/clique.py

```diff
@@ -36,8 +36,10 @@ class _QubitCouplingComposite(dimod.ComposedSampler):
        sampler (:obj:`~dimod.ComposedSampler`):
             A dimod sampler.
     """
-    def __init__(self, child_sampler):
+    def __init__(self, child_sampler, linear_range, quadratic_range):
```

Comment:
New args should probably be documented, for consistency, since we already have docs on this private class.